app/controllers/application_controller.rb

```diff
@@ -103,6 +103,10 @@ def current_live_form
     @current_live_form ||= Form.find_live(params[:form_id])
   end
 
+  def current_archived_form
```

Comment:
Move this into the ArchivedController (the others in here for example `current_live_form` if that can move, put that into the 2 places it's used).

Reply:
This is now moved as part of adding the archived controller